DNA-RNAclasses.py

```diff
@@ -4,7 +4,7 @@
 # In[ ]:
 
 
-#Dna class. Inherited from str slass.
+# Dna class. Inherited from str class.
 class Dna(str):
```

Comment:
What do you think about adding here a check of input type, for example you can make it with  ``assert isinstance(seq, str)``

Reply:
I see no reason for that - if its not string in the input for Dna - the TypeError will arise during initialization and so be it. However, the class I wrote should work if initialized from any string.
Also its 40 minutes till deadline and I'm not in a position to learn the usage of some new python code I never met before =)